packages/lib/utils/joplinCloud/accountTypeToString.ts

```diff
@@ -0,0 +1,15 @@
+enum AccountType {
+	Default = 0,
+	Basic = 1,
+	Pro = 2,
+	Team = 3,
+}
```

Comment:
This AccountType could be used elsewhere, including in joplinCloud.ts so I think this file should be named more generically. Maybe just types.ts.

Even `accountTypeToString` is type-related since it converts from one type to another